client/src/components/Bio/Bio.css

```diff
@@ -11,11 +11,18 @@
   color: var(--purple);
 }
 
-.icons__div div {
-  color: var(--purple);
+.Bio .icons__div-edit {
```

Comment:
I don't see these classes in your JSX. Can you remove them?

Unless these are classes in another component. You should never overwrite stiles in a child component! That will make debugging the CSS for that component nearly impossible in the future.

Reply:
>  You should never overwrite stiles in a child component!
I'm just using what I was given...  

I use :
.icons__div-edit is used in line 221 of Bio.js.


.icons__div div  is not mine.  It is called elsewhere on the Bio.js page.  I'm guessing someone previous used it to fix a styling bug that crops up with the TextField.  I can refactor the Bio.js page if you want me to, but I think others in the cohort are using it right now as well.  I tried to keep my footprint light.